src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/ApplyEditsToPreviousCSharpDoc_EnsureAccurateDeletionEdits.semanticexpected.txt

```diff
@@ -1,13 +1,13 @@
 ﻿//line,characterPos,length,tokenType,modifier
 0 0 4 15 0 //keyword
-0 5 1 30 0 //method name
-0 1 1 36 0 //punctuation
-0 1 1 36 0 //punctuation
-1 0 1 36 0 //puncutation
+0 5 1 35 0 //method name
```

Comment:
the TokenType values have changed here, but the description of them have not. Consulting the Legend I can see that the new way is correct.